src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerTransferEvidenceNoteTests.cs

```diff
@@ -0,0 +1,260 @@
+                .BeDecoratedWith<HttpPostAttribute>();
+        }
+
+        [Fact]
+        public void TransferPost_ShouldHaveAntiForgeryAttribute()
+        {
+            typeof(ManageEvidenceNotesController).GetMethod("Transfer", new[] { typeof(Guid) }).Should()
+                .BeDecoratedWith<ValidateAntiForgeryTokenAttribute>();
+        }
+
+        [Fact]
+        public void TransferEvidenceNoteGet_ShouldHaveHttpGetAttribute()
+        {
+            typeof(ManageEvidenceNotesController).GetMethod("TransferEvidenceNote", new[] { typeof(Guid) }).Should()
+             .BeDecoratedWith<HttpGetAttribute>();
+        }
+
+        [Fact]
+        public void TransferEvidenceNotePost_ShouldHaveHttpPostAttribute()
+        {
+            typeof(ManageEvidenceNotesController).GetMethod("TransferEvidenceNote", new[] { typeof(TransferEvidenceNoteDataViewModel) }).Should()
+             .BeDecoratedWith<HttpPostAttribute>();
+        }
+
+        [Fact]
+        public void TransferEvidenceNotePost_ShouldHaveAntiForgeryAttribute()
+        {
+            typeof(ManageEvidenceNotesController).GetMethod("TransferEvidenceNote", new[] { typeof(TransferEvidenceNoteDataViewModel) }).Should()
+                .BeDecoratedWith<ValidateAntiForgeryTokenAttribute>();
+        }
+
+        [Fact]
+        public void TransferPost_PageRedirectsToTransferPage()
+        {
+            var result = ManageEvidenceController.Transfer(OrganisationId) as RedirectToRouteResult;
+
+            result.RouteValues["action"].Should().Be("TransferEvidenceNote");
+            result.RouteValues["controller"].Should().Be("ManageEvidenceNotes");
+            result.RouteValues["organisationId"].Should().Be(OrganisationId);
+        }
+
+        [Fact]
+        public async Task TransferEvidenceNoteGet_GivenValidOrganisation__BreadcrumbShouldBeSet()
+        {
+            // arrange 
+            var organisationName = "OrganisationName";
+
+            A.CallTo(() => Cache.FetchOrganisationName(A<Guid>._)).Returns(organisationName);
+
+            // act
+            await ManageEvidenceController.TransferEvidenceNote(OrganisationId);
+
+            // assert
+            Breadcrumb.ExternalOrganisation.Should().Be(organisationName);
+            Breadcrumb.ExternalActivity.Should().Be(BreadCrumbConstant.SchemeManageEvidence);
+        }
+
+        [Fact]
+        public async Task TransferEvidenceNoteGet_GivenANewModelIsCreated_ModelWithDefaultValuesIsCreated()
+        {
+            // act
+            var result = await ManageEvidenceController.TransferEvidenceNote(OrganisationId) as ViewResult;
+            var model = result.Model as TransferEvidenceNoteDataViewModel;
+
+            // assert
+            model.CategoryValues.Should().AllBeOfType(typeof(CategoryBooleanViewModel));
+            model.CategoryValues.Count.Should().Be(Enum.GetNames(typeof(WeeeCategory)).Length);
```

Comment:
I'd do a loop through the WeeeCategoryEnum and check that all values are present in CategoryValues. Or do a linq query to check all enum values are present.